### ISSUE TITLE
Stage 2 learning rate

### ISSUE THREAD

**Issue by LeiFan95** [XiaomiMiMo/MiMo-VL#29]
Hi authors, thanks for the great work.

From the paper, I notice you might use a higher learning rate for ViT (1e-4) compared to the adapter (1e-5) during stage 2 training. Since ViT is already well-trained, I wonder about the reasons for using this setup.

Thanks.

**Comment by RenShuhuai-Andy** [comment]
Sorry we made a mistake in the technical report. In stage 2, the lr for ViT is 1e-5, and the lr for the projector is 1e-4.
